UnitTests/GitUITests/UserControls/SortDiffListContextMenuItemTests.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using GitCommands;
+using GitUI.UserControls;
+using NSubstitute;
+using NUnit.Framework;
+
+namespace GitUITests.UserControls
+{
+    [TestFixture]
+    public class SortDiffListContextMenuItemTests
+    {
+        private SortDiffListContextMenuItem _itemUnderTest;
+        private IDiffListSortService _testingSortService;
+
+        private string _originalTranslation;
+
+        [OneTimeSetUp]
+        public void OneTimeSetup()
+        {
+            _originalTranslation = AppSettings.CurrentTranslation;
+            AppSettings.CurrentTranslation = "en";
```

Comment:
I think you were looking for these attributes:
```
    [SetCulture("en-US")]
    [SetUICulture("en-US")]
    [TestFixture]
    public class SortDiffListContextMenuItemTests
    {
```

Reply:
I copied that from another test fixture. I can't remember which one now. If that ensures that the text is english then that is much better. Thanks for the tip.